tests/unit/containers/tutor-home-page/general-info-step/GeneralInfoStep.spec.jsx

```diff
@@ -0,0 +1,26 @@
+import { renderWithProviders } from '~tests/test-utils'
+import GeneralInfoStep from '~/containers/tutor-home-page/general-info-step/GeneralInfoStep'
+import { screen } from '@testing-library/react'
```

Comment:
Import libraries first
```suggestion
import { screen } from '@testing-library/react'
import { renderWithProviders } from '~tests/test-utils'
import GeneralInfoStep from '~/containers/tutor-home-page/general-info-step/GeneralInfoStep'

```